core/app/models/spree/order_updater.rb

```diff
@@ -226,7 +226,8 @@ def update_order_promotions
     end
 
     def update_taxes
-      Spree::Config.tax_adjuster_class.new(order).adjust!
+      output = Spree::Config.tax_calculator_class.new(order).calculate
+      Spree::Tax::Application.new(order, output).perform!
```

Comment:
If we move this change into the `Spree::Tax::OrderAdjuster`, we can keep overrides of that working (and delete the existing code in there)